_ideas/heatmap/intermediate/aggregate-discussions.md

```diff
@@ -0,0 +1,41 @@
+## Heatmap - Intermediate
+
+## Task
+Aggregate discussion from all of your courses instead of one course.
+
+## Overview
+Right now you've hard-coded your Canvas course id in [server](../../../backend/server.js) in the call to `getDiscussions`. Instead of hard-coding this course id, we'll use [`getCoursesByUser`](https://github.com/ubccapico/node-canvas-api/blob/master/src/getCoursesByUser.js) to retrieve all of the Canvas courses you have access to. `getCoursesByUser` takes as input a Canvas user id, which you can hard-code or be extra fancy about and call `getSelf` to retrieve (this is up to you). 
```

Comment:
No longer hard-coded Canvas course id -- in [5-Create Course-Dropdown.md](https://github.com/UBC-LA-Hackathon/student-dashboard/blob/master/5-Create-Course-Dropdown.md) we now already suggest students use`getCoursesByUser` to populate dropdown with course names.

Reply:
Good catch, I removed the reference